docs/userguide/format_parse/messages/mf2.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: default
+title: MessageFormat 2.0
+nav_order: 2
+parent: Formatting Messages
+grand_parent: Formatting
+---
+<!--
+© 2023 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# MessageFormat 2.0
+{: .no_toc }
+
+## Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+*Note: This page describes `MessageFormatter`, which is a Technical Preview API implementing
+MessageFormat 2.0.
+It will be a successor to the current [ICU MessageFormat](index.md).
+MessageFormat 2.0 is being developed
+[in a working group](https://github.com/unicode-org/message-format-wg),
+which has created a [draft of a specification](https://github.com/unicode-org/message-format-wg/tree/main/spec).
+Also see the 
+[API docs for `MessageFormatter`](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/index.html?com/ibm/icu/message2/MessageFormatter.html).*
+
+## Overview of `MessageFormatter`
+
+MessageFormatter is the next iteration of [MessageFormat](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/MessageFormat.html).
+This new version builds on what we learned from using MessageFormat for 20 years in various environments, either exposed "as is" or as a base for other public APIs.
+
+It is more modular, easier to backport, and provides extension points to add new formatters and selectors without having to modify the specification.
+
+We will be able to add formatters for intervals, relative times, lists, measurement units, people names, and more, and support custom formatters implemented by developers outside of ICU itself, for company or even product specific needs.
+
+MessageFormat 2 will support more complex grammatical features, such as gender, inflections, and tagging parts of the message for style changes or speech.
```

Comment:
I don't know how to phrase this better, but my mental model is that MF2 will _support supporting_ these more complex features. The support will not be built-in. I think it's important to be precise here so that the document doesn't make promises we can't keep.

Perhaps something to the effect of: `will enable building localization systems which support ...`?

Reply:
For line 40 I'd probably suggest:

> MF2 will provide the ability to support custom formatters beyond those in the MF2 specification. These will eventually include support for the additional formatters in ICU, such as intervals, relative time, lists, measurement units, personal names, and more, as well as the ability for users to supply their own custom implementations.

For line 42 I agree with @stasm. Perhaps:

> This includes, potentially, support for gender, inflection, markup regimes (such as those require for text-to-speech), and other complex message management needs.